src/Native/Testable/Navigation.js

```diff
@@ -0,0 +1,110 @@
+if (typeof _elm_lang$navigation$Native_Navigation === 'undefined') { // eslint-disable-line camelcase
+  throw new Error('Native.Testable.Navigation was loaded before _elm_lang$navigation$Native_Navigation: this shouldn\'t happen because Testable.Navigation imports Navigation. Please report this at https://github.com/avh4/elm-testable/issues')
+}
+
+var original_getLocation = _elm_lang$navigation$Native_Navigation.getLocation;
+_elm_lang$navigation$Native_Navigation.getLocation = function () {
+  if (typeof document !== 'undefined' && typeof document.location !== 'undefined') {
+    return original_getLocation()
+  }
+
+  return {
+    href: 'https://elm.testable/',
+    host: 'elm.testable',
+    hostname: 'elm.testable',
+    protocol: 'https:',
+    origin: 'https://elm.testable',
+    port_: '',
+    pathname: '/',
+    search: '',
+    hash: '',
+    username: '',
+    password: ''
+  }
+}
+
+var mapNavigation = function(ctor) {
+  return function (data) {
+      return {
+        ctor: 'Navigation_NativeNavigation',
+        _0: {
+          ctor: ctor,
+          _0: data
+        },
+        _1: function (data) { return { ctor: 'Success', _0: data } }
+      }
+  }
+}
+
+_elm_lang$navigation$Navigation$pushState = setItUp(
```

Comment:
It doesn't, I don't remember exactly why right now, maybe loading order or idk, I was using this at first but had to overwrite both anyway so I got rid of Native_Navigation in [this refactor](https://github.com/rogeriochaves/elm-testable/commit/7b1f64aa31662308c23168f7a28e94189046cd97). Maybe we should try it again later.